moved footer to bottom and put a border around sign in

jkasato opened the pull request:
moved footer to bottom and put a border around sign in

jkasato committed:
moved footer to bottom and put a border around sign in

moved footer to bottom and put a border around sign in

mrobertson-csumb reviewed (changes requested):
The files in the bin directory should not have been added

jkasato committed:
css changes

modified font size for h1 and h1
removed margins from header and navbar
removed rounded edges from jumbotron

jkasato commented:
> The files in the bin directory should not have been added

How do I removed those changes and only include the css file that I added?

jkasato committed:
add bin to gitignore

jkasato committed:
bin

mrobertson-csumb commented on .gitignore:
```diff
@@ -2,6 +2,7 @@ target/
 !.mvn/wrapper/maven-wrapper.jar
 !**/src/main/**
 !**/src/test/**
+!**/bin/**
```
this line as written is an unignore, if it stays this way, `git add .` will re-add all of bin's contents.
it should simply be:
bin/

mrobertson-csumb commented on src/main/resources/public/css/styles.css:
```diff
@@ -66,33 +74,54 @@ ul.nav li a:visited,
 ul.nav li a:hover,
 ul.nav li a:active,
 ul.nav li.active a {
+	padding:0;
```
use a code formatter on the file before committing

mrobertson-csumb commented on src/main/resources/public/css/styles.css:
```diff
@@ -66,33 +74,54 @@ ul.nav li a:visited,
+    position: absolute;
+ 	bottom: 0;
+  	width: 100%;
+  	height: 3 rem; /* Footer height */
```
no space between scalar and unit

mrobertson-csumb commented on src/main/resources/public/css/styles.css:
```diff
@@ -66,33 +74,54 @@ ul.nav li a:visited,
+}*/
+
+
+#footer-id {
```
footer is unreadably small when rendered on the page.
footer is not anchored at the very bottom of the page, there is a gap about 1.5x the height of the footer.

mrobertson-csumb reviewed (changes requested):
Needs minor changes.
See inline comments.

jkasato committed:
addressed PR issues

addressed PR issues

mrobertson-csumb commented on src/main/resources/public/css/styles.css:
```diff
@@ -1,130 +1,155 @@
+	bottom: 0px;
+	height: 3rem; width : 100%;
+	border: 3px solid black;
+	width: 100%;
```
width already defined above.

mrobertson-csumb commented on src/main/resources/public/css/styles.css:
```diff
@@ -1,130 +1,155 @@
+#footer-id {
+	background: black;
+	text-align: center;
+	font-size: 0.6em;
```
0.8em is the smallest that looks OK on my browser.

jkasato committed:
Update styles.css

0.6em -> 0.8em
position absolute-> position fixed
removed second width value
